guide/popular-topics/reloading-commands.md

```diff
@@ -0,0 +1,62 @@
+# Reloading Commands
+
+When writing your commands, you may find it tedious to restart your bot every time for testing the smallest changes, With a command handler, you can eliminate this issue and reload your commands while your bot is running.
+
+This section assumes you did follow the [Command Handling](/guide/creating-your-bot/command-handling.md) part.
```

Comment:
```suggestion
This section assumes you followed the [Command Handling](/guide/creating-your-bot/command-handling.md) part.
```

Reply:
"the Command Handling part" doesn't seems exactly right, what are the thoughts on "the Command Handling section", or maybe the standard TIP could be better here "This page is a follow-up to the Command Handling page. Please carefully read it first so that you can understand the methods used in this section."

Reply:
"section" sounds good as does the generic tip.